src/index.js

```diff
@@ -5,8 +5,8 @@ const { move } = require('./robot');
 
 const villageMap = buildVillageMap(roads);
 let randomLocation = _.sample(getStates(villageMap));
-for (let i = 1; i <= roads.length; i++) {
+_.range(10).forEach((i) => {
```

Comment:
out of curiosity, why do you use `_.range`?
you could've achieved the same by `roads.forEach((_v, index) => {})`